pkg/graph/schema.resolvers.go

```diff
@@ -489,25 +489,34 @@ func (r *mutationResolver) CreateAccessibilityRequest(ctx context.Context, input
 		return nil, err
 	}
 
-	intake, err := r.store.FetchSystemIntakeByID(ctx, input.IntakeID)
-	if err != nil {
-		return nil, err
+	if input.IntakeID == nil && (input.CedarSystemID == nil || len(*input.CedarSystemID) == 0) {
+		return nil, errors.New("Either a system intake ID or a CEDAR system ID is required to create a 508 request")
 	}
 
 	newRequest := &models.AccessibilityRequest{
 		EUAUserID: requesterEUAID,
 		Name:      input.Name,
-		IntakeID:  input.IntakeID,
+	}
+
+	var applicationName string
+	if input.IntakeID != nil {
+		intake, err2 := r.store.FetchSystemIntakeByID(ctx, *input.IntakeID)
```

Comment:
Can you name this err more specifically for the fact that it relates to fetching system intakes?